chsdi/templates/htmlpopup/base.mako

```diff
@@ -53,7 +53,7 @@
 <body>
   <div class="chsdi-htmlpopup-container">
 % else:
-  <div class="htmlpopup-container">
+  <div id="${c['layerBodId']}#${c['featureId']}" class="htmlpopup-container">
```

Comment:
The issue was with these primary keys which were changed with every update (`non stabile ids`)